### PR TITLE
feat: update latest/beta based on 1.8/beta

### PR DESCRIPTION
**Pull request by NohaIhab**
Updates the `latest/beta` bundle definition due to changes made in `1.8/beta`

### CODE REVIEW

File: releases/latest/beta/bundle.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ applications:
     _github_repo_name: admission-webhook-operator
   argo-controller:
     charm: argo-controller
-    channel: latest/beta
+    channel: 3.3/stable
```
Comment (DnPlas) [inline releases/latest/beta/bundle.yaml:13]: I suggest we leave this as latest/beta. [This PR](https://github.com/canonical/argo-operators/pull/141) should fix the issue we observed during our testing phase, and the fix is now in both latest/edge and latest/beta.

**Review by DnPlas — COMMENTED** [review]
Thanks @NohaIhab, small comment.

File: releases/latest/beta/bundle.yaml (hunk 1 above)
Comment (NohaIhab) [reply releases/latest/beta/bundle.yaml:13]: agreed. based on this, should we also update it in 1.8/beta?

Comment (DnPlas) [reply releases/latest/beta/bundle.yaml]: I'd say so, just so we don't forget.

**Review by DnPlas — APPROVED** [review]
Thanks @NohaIhab, please merge and let's trigger the publish action.

**Pull request merged**